client/layout/guided-tours/docs/API.md

```diff
@@ -216,3 +216,53 @@ combineTours( {
   thirdTour: ThirdTour,
 } );
 ```
+
+## Targeting elements in Calypso
+
+In few places in GT you need to target a DOM elements in Calypso. There are currently two ways to do this:
```

Comment:
To explain myself a little better: assuming the author has read the rest of the docs — notably, Getting Started / Tutorial — they'll know that they can target elements. I'm not suggesting we explain that all over again, just that we mention that concept to introduce the section on how to reference those targets.